src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AppOfflineHandler.h

```diff
@@ -4,15 +4,16 @@
 #pragma once
 
 #include <string>
+#include <utility>
 #include "requesthandler.h"
 
 class AppOfflineHandler: public REQUEST_HANDLER
 {
 public:
-    AppOfflineHandler(IHttpContext& pContext, const std::string appOfflineContent)
+    AppOfflineHandler(IHttpContext& pContext, std::string appOfflineContent)
```

Comment:
This was probably supposed to be `const std::string&` to avoid the unneeded copy.

Reply:
Yeah, I was getting a warning as well, but I have fixed this with `std::move`, because of which it won't create an unnecessary copy.

Reply:
There is still an unneeded copy from the callsite of this constructor

Reply:
OK, I will get this checked!